demo/assets/js/musicplayer.js

```diff
@@ -0,0 +1,254 @@
+            }
+        }, 1000);
+	}
+
+    /**
+    *  input music index, then player run
+    *
+    *  @param {Number} index
+    */
+    playMusic(index) {
+        // if index passed, it means play specific music
+        if(typeof index === 'number') {
+            this.bulletHandler(this.currentMusicIndex).inactive();
+            this.currentMusicProgress = 0;
+            this.currentMusicIndex = index;
+        }
+        this.player(this.currentMusicIndex);
+        this.turnTableDom.classList.remove('paused');
+    }
+
+    // pause player
+	pauseMusic() {
+        clearInterval(this.timer);
+        this.turnTableDom.classList.add('paused');
+    }
+
+    // play next music
+	nextMusic() {
+        // if last music of list, go to the first music
+		if (this.currentMusicIndex >= this.playList.length - 1) {
+            this.bulletHandler(this.currentMusicIndex).inactive();
+			this.currentMusicIndex = 0;
+		} else {
+            this.bulletHandler(this.currentMusicIndex).inactive();
+			this.currentMusicIndex = ++this.currentMusicIndex;
+		}
+		this.playMusic();
+	}
+
+    // play previous music
+	prevMusic() {
+		if (this.currentMusicIndex <= 0) {
+            this.bulletHandler(this.currentMusicIndex).inactive();
+			this.currentMusicIndex = this.playList.length;
+		} else {
+            this.bulletHandler(this.currentMusicIndex).inactive();
+			this.currentMusicIndex = --this.currentMusicIndex;
+		}
+		this.playMusic();
+	}
+
+    bulletHandler(index) {
+        return {
+            active: () => {
+                this.bulletContainerDom.querySelectorAll(".bullet")[index].classList.add('active');
+            },
+            inactive: () => {
+                this.bulletContainerDom.querySelectorAll(".bullet")[index].classList.remove('active');   
+            }
+        }
+    }
+
+    // handle list using egjs-axes
+    listHandler() {
+        let uinitDeg =  360 / this.playList.length; // 단위각
+        let unitRad = uinitDeg * Math.PI / 180; // 단위라디안
+        let rangeRad = unitRad / 2; // 범위라디안
+        let destMusicIndex; // 라디안 계산에 의해 실행될 음악
+        let translateZ =  Math.round( ( 210  / 2 ) /  Math.tan( Math.PI / this.playList.length ));
+        
+        let axes = new eg.Axes({
+            axis: {
+                rotateX: {
+                    range: [0, 360],
+                    circular: true
+                },
+            },
+            deceleration: 0.0024
+        }).on({
+            "change": (e) => {
+                this.listCarouselDom.style[eg.Axes.TRANSFORM] = "translateZ("+ -1 * translateZ +"px) rotateY(" + e.pos.rotateX + "deg)";
+            },
+            "hold": (e) => {
+                this.pauseMusic();
+            },
+            "release": (e) => {
+                destMusicIndex = 0;
+                let currentRadian = e.destPos.rotateX * Math.PI / 180; // 현재라디안
+
+                // 자기자신의 범위라디안에 들어온 뮤직을 가운데로 위치
+                for(let i = 0; i < this.playList.length; i++) {
```

Comment:
you should check e.destPos.
if destPos is not equal depaPos you should pass this job to `animationEnd`.